install.sh

```diff
@@ -789,10 +789,11 @@ install_file() {
       fi
 
       rpm -Uvh --oldpackage --replacepkgs "$2"
+      exists dnf && PKGCMD=dnf || PKGCMD=yum
```

Comment:
For consistency with other usages of `exists` like https://github.com/puppetlabs/install-puppet/blob/849de12b07bc4bd4d542c447a96ffb1ecc0cfc45/install.sh#L557-L561 could you expand this
```suggestion
      if exists dnf; then
        PKGCMD=dnf
      else
        PKGCMD=yum
      fi
```